pyqtgraph/graphicsItems/PlotItem/PlotItem.py

```diff
@@ -240,13 +240,13 @@ def __init__(self, parent=None, name=None, labels=None, title=None, viewBox=None
         self.ctrl.avgParamList.itemClicked.connect(self.avgParamListClicked)
         self.ctrl.averageGroup.toggled.connect(self.avgToggled)
         
-        self.ctrl.maxTracesCheck.toggled.connect(self.updateDecimation)
+        self.ctrl.maxTracesCheck.toggled.connect(self._max_trace_wrapper_slot)
         self.ctrl.forgetTracesCheck.toggled.connect(self.updateDecimation)
         self.ctrl.maxTracesSpin.valueChanged.connect(self.updateDecimation)
         
         if labels is None:
             labels = {}
-        for label in list(self.axes.keys()):
+        for label in list(self.axes.keys()):b
```

Comment:
looks like a typo resulting in a syntax error?

Reply:
yes :/ navigating with thinkpad trackpoint...